src/main.js

```diff
@@ -191,3 +194,29 @@ export const runPush = async () => {
         prevCommit = commitData.data.sha;
     }
 };
+
+export type __TestCommit = {
+    author: '__testAuthor',
+    comment_count: -1,
+    committer: '__testCommitter',
+    id: string,
+    message: string,
+    tree: '__TESTING__',
+    url: '__TESTING__',
+    verification: '__TESTING__',
+};
+
+type __TestContext = {|
```

Comment:
Is this extra type needed? It doesn't seem like it should be - the input should be about to be the same type as the Context, so we can remove this type and the `__TestCommit` type.